flipperdevices/flipperzero-firmware
# [FL-1171] Remove core-rs

# What's new
Remove core-rs and rust syntax check

# Verification

- Build for f4 and f5 targets
- Verify that nothing has been broken

# Checklist (do not modify)
